### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by maxplooi** [visualdesigncafe/nature-renderer#1] [General]
### Welcome to the official Discussions forum for Nature Renderer!
I have opened this forum to bring the community together and provide a better alternative for support/feedback than email. As a solo-developer it is becoming increasingly difficult to provide personal email support for everyone with the ever-growing user base of Nature Renderer, so I hope we can all come together and help each other out in these forums.

#### Categories
- 📣 [Announcements](https://github.com/visualdesigncafe/nature-renderer/discussions/categories/announcements) - Announcements for new releases and other news.
- 🙏 [Help](https://github.com/visualdesigncafe/nature-renderer/discussions/categories/help) - To get help with Nature Renderer.
- 💡 [Ideas ](https://github.com/visualdesigncafe/nature-renderer/discussions/categories/ideas)- For providing feedback, feature requests, and other ideas to improve Nature Renderer.
- 💬 [General ](https://github.com/visualdesigncafe/nature-renderer/discussions/categories/general)- For general discussions that don't fit in any other category.

#### Other Resources
- 🔗 [Issue Tracker](https://github.com/visualdesigncafe/nature-renderer/issues/) - Create and browse bug reports
- 🔗 [Documentation](https://visualdesigncafe.com/nature-renderer/docs/) - The official documentation on how to use Nature Renderer

**Comment by xl412cs** [comment]
How can use it in webgl?

**Reply by maxplooi** [reply]
WebGL is not support due to limitations of the platform.
